apps/api/src/app/widgets/dtos/feeds-response.dto.ts

```diff
@@ -78,7 +78,7 @@ export class NotificationFeedItemDto implements INotificationDto {
     example: 'feed_445566',
     type: String,
   })
-  _feedId: string;
+  _feedId?: string | null;
```

Comment:
We would need to update the ApiProperty above, asd it's the one who is resposinble for the SDK gen, we can use the ApiPropertyOptional decorator for that and adding a nullable: true to it

Reply:
Thanks for the quick reply, I updated it :)